projects/rpc-semconv.md

```diff
@@ -0,0 +1,67 @@
+
+### Industry outreach
+
+- @JamesNK from .Net gRPC library - open to us reaching out as needed
+
+### Required staffing
+
+Projects cannot be started until the following participants have been identified:
+* Every project needs a project lead, who is willing to bottom line the project and address any issues which are not handled by other project members.
+* At least two sponsoring [TC](../community-members.md#technical-committee) or [GC](../community-members.md#governance-committee) members (or [semantic convention maintainers](https://github.com/orgs/open-telemetry/teams/specs-semconv-maintainers) in the case of semantic convention SIGs), or community members delegated by them. Sponsors are dedicated to attending meetings, reviewing proposals, and in general being aware of the state of the project and its technical details. Sponsors guide the project through the spec process, keep the tracking issue up to date, and help to ensure that relevant community members provide input at the appropriate times.
+* A GC liaison to facilitate this SIG's health and ensure project scope remains true to the project description. If a GC member is also a sponsor for this project, they are by default the GC liaison (see [GC check-ins](../gc-check-ins.md)).
+* Engineers willing to write prototypes in at least two languages (if relevant to project). Languages should be fairly different from each other (for example: Java and Python).
+* Maintainers or approvers from those languages committed to reviewing the prototypes.
+
+## Timeline
+
+Starting in mid-July. Targeting completion by KubeCon North America 2025.
+
+## Labels
+
+[`area:rpc`](https://github.com/open-telemetry/semantic-conventions/issues?q=state%3Aopen%20label%3A%22area%3Arpc%22)
+in the semantic-conventions repository.
+
+## Project Board
+
+Once approved, a project should be managed using a GitHub project board (see [open projects](https://github.com/orgs/open-telemetry/projects?query=is%3Aopen)). This project board should be pre-populated with issues that cover all known deliverables, organized by timeline milestones.
+
+Any [member](../guides/contributor/membership.md) associated with the project can create the board. Once created, the creator of the board should:
+
+- Assign `Admin` privileges on the project to the relevant project members (using a new or existing GitHub team).
+- Change the visibility of the project to `Public` in order to share project status and priorities outside of the OpenTelemetry organization.
+- Configure project workflows to automatically add issues and PRs to the board based on repositories and labels.
+
+Once created, please add a link to the project board here.
+
+## SIG Meetings and Other Info
+
+Once a project is started, its corresponding SIG should meet regularly for discussion. These meeting times should be posted on the [OpenTelemetry public calendar](https://github.com/open-telemetry/community#calendar) and automatically recorded.
```

Comment:
Created the doodle - https://doodle.com/group-poll/participate/dGWjjQ0d/vote. Soft plan to start meeting in the second week of July.

 @trask, @mconnew, @albumenj, @SteveRao please vote!

Reply:
Updated poll - https://strawpoll.com/kjn1Dmoz0yQ

@trask, @mconnew, @AlbumenJ, @steverao please vote!